documentation/conda.yml

```diff
@@ -0,0 +1,14 @@
+# Python environment and dependencies to build the documentation
+
+name: cocotb-docs3
+dependencies:
+  - python=3.8
+  - pip
+  # ReadTheDoc dependencies
+  - mock
+  - pillow
+  - sphinx
+  - sphinx_rtd_theme
```

Comment:
```suggestion
  # ReadTheDoc dependencies
  - mock
  - pillow
  - sphinx
  - sphinx_rtd_theme
```

Reply:
@eric-wieser Should we try this?

Reply:
Am I blind, or is your suggestion a no-op?

Reply:
Now better?